### ISSUE TITLE
Author and  CreatedUtc  always  null, when using api 【1.7.0-preview-17548】

### ISSUE THREAD

**Issue by hyzx86** [OrchardCMS/OrchardCore#14091]
Hi , 
I recently upgraded the OC version of the project reference
 from    `1.7.0-preview-17487`   to `1.7.0-preview-17548`

And found that the `Author` and `CreatedUtc`  are always `null` when the content item is created or updated

What was recently updated that caused Author, CreatedUtc to always be empty when calling api to create content?

**Comment by hyzx86** [comment]
![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/af317014-5b99-4a4f-b674-4019358146fe)
![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/d91dd2da-5875-4e77-920d-1a2602954524)

**Comment by hishamco** [comment]
We need to check

**Comment by hishamco** [comment]
> Author and CreatedUtc always null, when using api 

Which API you are refer to?

**Comment by hyzx86** [comment]
I have found a solution to this problem, but for some reason, 

I copied a `DefaultContentManager` class and then overrode the default one in my project. 
added breakpoints in the constructor to monitor the `IContentHandler` collection, and at that point, 
I noticed that `UpdateContentsHandler` was not being loaded. 
So, I re-registered `UpdateContentsHandler` in my project and the problem was resolved. 
However, I cannot be sure if any other `IContentHandler`s were lost.

**Comment by hyzx86** [comment]
> > Author and CreatedUtc always null, when using api
> 
> Which API you are refer to?

Not only content api ，In my subsequent testing, I found that submitting directly through the MVC page also had this problem

**Comment by hyzx86** [comment]
version : 1.7.0-preview-17427

![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/d670f373-7320-4f87-aa85-bd345beea396)

**Comment by MikeAlhayek** [comment]
@hyzx86 I can't reproduce the issue. `UpdateContentsHandler` is registered using the `AddContentManagement()` extension and the create info are set there. I am guessing that you have custom handler or code in your project that is throwing exception which causes the `UpdateContentsHandler` not to get called. 

Can you please check your log file and see if there is a silent unhanded exception?

**Comment by hyzx86** [comment]
@MikeAlhayek  Yes ，Indeed， I did add a few handlers, but notice the variable monitor area expression in the screenshot above

Different versions of expressions compute different results on the same code
```
Handlers.FirstOrDefault(x=>x.GetType().Name=="UpdateContentsHandler")
```

the property `Handlers`  is initialized in  constructor

**Comment by MikeAlhayek** [comment]
If the `OrchardCore.Contents` feature is not enabled, that would return null.

**Comment by hyzx86** [comment]
> If the `OrchardCore.Contents` feature is not enabled, that would return null.

Yes, I'm sure it's enabled, otherwise I wouldn't have been able to access the content management page

**Comment by MikeAlhayek** [comment]
Maybe somehow you are unregistered the handler in your project? I can't reproduce the issue in main branch. Can you try to reproduce the issue with a fresh outside the project?

**Comment by hyzx86** [comment]
On the left side are the Handlers when the content list is loaded

On the right are the Handlers that are loaded when content items are distributed, 
and they have the same number of Handlers, 
but they have different content, 



![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/4b7dd3c9-227f-47a8-bb0d-b47ff88f2176)


What would affect this list?

Currently 1.7.0-preview-17487 is used

It does seem to be a problem with my code, but I can't find the problem.

**Comment by hyzx86** [comment]
Problem solved

![image](https://github.com/OrchardCMS/OrchardCore/assets/15613121/5795d9f7-b5be-48c7-a28c-644fee1263f5)
